### DISCUSSION TITLE
renovate/renovate:37-full seems to ignore

### DISCUSSION THREAD

**Discussion by francisATgwn** [renovatebot/renovate#29844] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

I'm trying to spin up renovate via docker for a proof of concept. It complains with `FATAL: You must configure a GitHub token` despite what I think is correct configuration.

Invocation:

```
docker run --rm -it --env-file renovate.env --env-file secret.env renovate/renovate:37-full
```

renovate.env:
```
MEND_RNV_ACCEPT_TOS=y
MEND_RNV_PLATFORM=gitlab
MEND_RNV_ENDPOINT=https://********/api/v4/
MEND_RNV_SERVER_PORT=9000
MEND_RNV_SQLITE_FILE_PATH=/db/renovate-ce.sqlite
MEND_RNV_AUTODISCOVER_FILTER=renovate-poc/*
LOG_LEVEL=debug
```

secret.env:
```
MEND_RNV_GITLAB_PAT=glpat-********
MEND_RNV_WEBHOOK_SECRET=********
GITHUB_COM_TOKEN=github_pat_*******
```

What am I doing wrong?

### Logs (if relevant)

<details><summary>Logs</summary>

```
DEBUG: Using RE2 regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in config.js
DEBUG: No config file found on disk - skipping
DEBUG: Converting GITHUB_COM_TOKEN into a global host rule
DEBUG: File config
       "config": {}
DEBUG: CLI config
       "config": {}
DEBUG: Env config
       "config": {
         "hostRules": [
           {"hostType": "github", "matchHost": "github.com", "token": "***********"}
         ]
       }
DEBUG: Combined config
       "config": {
         "hostRules": [
           {"hostType": "github", "matchHost": "github.com", "token": "***********"}
         ]
       }
DEBUG: Enabling forkProcessing while in non-autodiscover mode
DEBUG: Found valid git version: 2.45.2
DEBUG: Setting global hostRules
DEBUG: Adding token authentication for github.com (hostType=github) to hostRules
FATAL: You must configure a GitHub token
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 60,
           "logContext": "ZUcH5Yw-BURPamOTt3UTb",
           "msg": "You must configure a GitHub token"
         }
       ]
```

</details>

**Comment by rarkins** [comment, marked as answer]
Configure RENOVATE_TOKEN

**Reply by francisATgwn** [reply]
Can you please point me to the documentation for that? I've been following https://github.com/mend/renovate-ce-ee/blob/main/docs/configure-renovate-ce-gitlab.md and it makes no mention of that env var.

I see it in https://docs.renovatebot.com/self-hosted-configuration/ but the description "Repository Auth Token" doesn't help me.

**Reply by viceice** [reply]
you're mixing the Mend Renovate CE/EE config with the open source Renovate CLI. you should follow that docs more, there are samples on that page

**Reply by rarkins** [reply]
You're following the wrong docs. You are using the Renovate CLI and not Renovate CE. Read https://docs.renovatebot.com/getting-started/running/#hosting-renovate instead

**Reply by francisATgwn** [reply]
Got it. Thank you both. I'll take a closer look at the docs.

**Reply by HonkingGoose** [reply]
I'm adding a section to the homepage to tell readers if they're in the right place:

- #30039
